3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune.Android/EnrollmentNotificationReceiver.cs

```diff
@@ -28,16 +30,34 @@ class EnrollmentNotificationReceiver : Java.Lang.Object, IMAMNotificationReceive
         /// </returns>
         public bool OnReceive(IMAMNotification notification)
         {
+            Debug.WriteLine("***Begin EnrollmentNotificationReceiver -> OnReceive");
```

Comment:
Why is the remapper needed? Is it only for CITI scenario? If so, since this is a public sample, can we put these changes into a dev app to not clutter the sample? Otherwise, I would clarify what the remapper does.
Similarly to the above, these log messages make it less clear on what's going on with the code.